<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicSourceUpdateWidget</class>
 <widget class="QDialog" name="MusicSourceUpdateWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>软件升级精灵</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QToolButton" name="topTitleCloseButton">
   <property name="geometry">
    <rect>
     <x>425</x>
     <y>10</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>454</width>
     <height>260</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#FCFCFC;</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="first">
    <widget class="QLabel" name="titleLable">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>454</width>
       <height>215</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size:15px; color:rgb(100, 100, 100);</string>
     </property>
     <property name="text">
      <string>天天酷音正在检测版本号...</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="upgradeButton">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>223</y>
       <width>88</width>
       <height>28</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{border-radius:2px;background-color:#158FE1;color:white;font-size:13px;}</string>
     </property>
     <property name="text">
      <string>立即升级</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cancelButton">
     <property name="geometry">
      <rect>
       <x>355</x>
       <y>223</y>
       <width>88</width>
       <height>28</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{border-radius:2px;background-color:#FCFCFC;color:#888888;font-size:13px;}</string>
     </property>
     <property name="text">
      <string>以后再说</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>215</y>
       <width>454</width>
       <height>45</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:#F7F7F7;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label</zorder>
    <zorder>titleLable</zorder>
    <zorder>upgradeButton</zorder>
    <zorder>cancelButton</zorder>
   </widget>
   <widget class="QWidget" name="second">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QLabel" name="timeLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>125</y>
       <width>201</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(100, 100, 100);</string>
     </property>
     <property name="text">
      <string>估计剩余时间: --</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="downProgressBar">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>415</width>
       <height>25</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="fileSizeLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>201</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(100, 100, 100);</string>
     </property>
     <property name="text">
      <string>文档大小: --</string>
     </property>
    </widget>
    <widget class="QLabel" name="speedLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>95</y>
       <width>201</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(100, 100, 100);</string>
     </property>
     <property name="text">
      <string>传送速率: --</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>454</width>
     <height>90</height>
    </rect>
   </property>
   <property name="pixmap">
    <pixmap resource="../../../../TTKUi/MusicPlayer.qrc">:/toolSets/lb_upgradeMain</pixmap>
   </property>
  </widget>
  <zorder>stackedWidget</zorder>
  <zorder>background</zorder>
  <zorder>topTitleCloseButton</zorder>
 </widget>
 <resources>
  <include location="../../../../TTKUi/MusicPlayer.qrc"/>
 </resources>
 <connections/>
</ui>
